src/components/BorrowerProfile/LendCta.vue

```diff
@@ -575,11 +549,7 @@ export default {
 		},
 	},
 	methods: {
-		async addToBasket(lendAmount = 0) {
-			if (lendAmount) {
-				this.$kvTrackEvent('Borrower profile', 'click', 'loan-activities-lend', this.loan?.id, lendAmount);
```

Comment:
Are we sure this event was specific to activity feed? Was it part of those PRs?

Reply:
Yeah, we added for this case only.